load_fixtures.py

```diff
@@ -0,0 +1,18 @@
+from os import chdir, getcwd, listdir, path, system
+
+original_directory = getcwd()
+apps = ['authentication', 'property', 'pages']
+files = []
+
+
+def loaddata(file):
+    if path.splitext(file)[1] == '.json' and file != 'initial_data.json':
+        print(file)
+        system("python3 manage.py loaddata %s" % file)
+
+
+for app in apps:
+    files = listdir(original_directory+'/'+app+'/fixtures')
```

Comment:
A more pythonic approach here would be to use [f-Strings](https://realpython.com/python-f-strings/) instead of concatenating with `+`. You can try that and see if it works.

```python
files = listdir(f'{original_directory}/{app}/fixtures')
```

Reply:
@wangonya thanks for this suggestion. it's such a cool approach which I had forgotten about.

Reply:
@wangonya kindly note that implemented the suggested approach. You may continue to review/approve.